source/github-helpers/load-details-menu.ts

```diff
@@ -2,7 +2,7 @@ import select from 'select-dom';
 import oneEvent from 'one-event';
 
 export default async function loadDetailsMenu(detailsMenu: HTMLElement): Promise<void> {
-	const fragment = select('include-fragment.SelectMenu-loading', detailsMenu);
+	const fragment = select('.js-comment-header-actions-deferred-include-fragment', detailsMenu);
```

Comment:
They changed classes, so if the user never hovered the dropdown for it to preload it, the action would just silently fail